class-udm-updater.php

```diff
@@ -66,6 +67,12 @@ public function __construct($mothership, $muid, $relative_plugin_file, $options
 		// Prevent updates from wordpress.org showing in all circumstances. Run with lower than default priority, to allow later processes to add something.
 		add_filter('site_transient_update_plugins', array($this, 'site_transient_update_plugins'), 9);
 
+		$this->udmupdaterl10n = array(
+			'duplicate_site_id1' => esc_js(__("Your attempt to connect to the updates server was unsuccessful due to duplicate site IDs. However, the server has disconnected the site that relates to the detected site ID. Please connect again to claim this site and to receive updates.", 'udmupdater')),
+			'duplicate_site_id2' => esc_js(__('This site was previously sharing a single licence also used another site (URL), probably because of being duplicated — this resulted in it being disconnected', 'udmupdater')),
```

Comment:
@andtown Looking further down this class, I see that this text - `duplicate_site_id2` - is being shown at the same time as `duplicate_site_id1`. That doesn't make sense to me, as the two messages say different things. (Lines 614 and 615).

Reply:
@DavidAnderson These two messages will now be separately shown as the way they are numbered (i.e. duplicate_site_id1 is the first message the users will see when they connect their cloned site to the updates server, and the second message will appear after the second attempt of connectivity)